src/Components/ActivationKeysDocsPopover/ActivationKeysDocsPopover.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import {
+  Button,
+  Popover,
+  PopoverPosition,
+  Text,
+  TextContent,
+} from '@patternfly/react-core';
+import propTypes from 'prop-types';
+import { OutlinedQuestionCircleIcon } from '@patternfly/react-icons';
+
+const ActivationKeysDocsPopover = (props) => {
+  const { orgId } = props;
+  return (
+    <Popover
+      headerContent="Activation Keys"
+      position={PopoverPosition.rightEnd}
```

Comment:
using `rightEnd` the popover gets positioned at the bottom due to spacing.  what about `rightStart` instead? that seems to place it to the right.